include/vsg/core/Array.h

```diff
@@ -33,7 +33,7 @@ namespace vsg
 {
 
     template<typename T>
-    class Array : public Data
+    class VSG_DECLSPEC Array : public Data
```

Comment:
If Array requires  VSG_DECLSPEC then Value, Array2D and Array3D will also require it.